.github/workflows/verify_rc.yml

```diff
@@ -234,13 +255,32 @@ jobs:
       VERSION: ${{ needs.target.outputs.version }}
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
       - name: Run
         run: |
           dev/release/verify-release-candidate.sh ${VERSION} ${RC}
       - name: Verify the previous version
+        # TODO: We may re-enable this in the future.
+        # There are some problems for now:
+        # * x86_64: libLLVM.so.18.1 needed by gandiva2000-libs on AlmaLinux 9
```

Comment:
This is not our problem. AlmaLinux 9 upgraded its LLVM version to 19 and stopped providing 18.